web/src/components/Korrel8rPanel.tsx

```diff
@@ -116,7 +126,14 @@ export default function Korrel8rPanel({ initialQueryString }: Korrel8rPanelProps
   return (
     <>
       <FlexItem className="tp-plugin__panel-query-container">
-        <Title headingLevel="h2">{t('Correlation Signals')}</Title>
+        <Title headingLevel="h2">{t('Correlated Signals')}</Title>
+        <Button
+          title="Find correlated signals from the resources on the current console page."
+          isAriaDisabled={!korrel8rQueryFromURL || korrel8rQueryFromURL == savedQuery}
+          onClick={() => { dispatchQuery(korrel8rQueryFromURL) }}
+        >
+          Sync
+        </Button>
```

Comment:
Rather than use a title, we should be using the Patternfly native [tooltip](https://www.patternfly.org/components/tooltip) component.

```suggestion
        <Tooltip content="Find correlated signals from the resources on the current console page.">
          <Button
            isAriaDisabled={!korrel8rQueryFromURL || korrel8rQueryFromURL == savedQuery}
            onClick={() => { dispatchQuery(korrel8rQueryFromURL) }}
          >
            Sync
          </Button>
        </Tooltip>
```